src/components/bcros/Address.vue

```diff
@@ -16,12 +22,15 @@
           {{ $t('label.address.addressType.mailing') }}
         </div>
         <BcrosAddressDisplay
-          v-if="differentMailingAddress(address)"
+          v-if="address?.mailingAddress && differentMailingAddress(address)"
           :address="address.mailingAddress"
         />
-        <div v-else>
+        <div v-else-if="address?.mailingAddress">
           {{ $t('text.general.sameAsAbove') }}
         </div>
+        <div v-else>
+          {{ $t('text.general.notEntered') }}
+        </div>
```

Comment:
If mailing address is null, show (Not entered).

Example: https://business-dashboard-dev--pr-183-7cbof13k.web.app/FM1130015